charts/immich/templates/server.yaml

```diff
@@ -58,8 +58,10 @@ probes:
       timeoutSeconds: 1
       failureThreshold: 3
   readiness: *probes
-  startup:
-    enabled: false
+  startup: *probes
+    spec:
```

Comment:
Best I can tell, this is not valid YAML, but somehow helm is letting it through anyways - but then the diff it produces is really weird & touches a bunch of unrelated stuff.

Reply:
https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

Reply:
This is how I've been using for some time, but couldn't find in the Helm documentation, so I fixed the PR.